rules/rbe_repo.bzl

```diff
@@ -784,6 +788,10 @@ _rbe_autoconfig = repository_rule(
                    "JAVA_HOME env var from the container. If that is not set, the rule " +
                    "will fail."),
         ),
+        "local_image": attr.string(
```

Comment:
Adding another attribute adds additional complexity to the rule, so we should carefully discuss the need to do so.

1. I was wondering if there was a way to check if there's a local image rather than do a pull unconditionally. Did you consider such an option?
2. Maybe it would be sufficient to have a boolean attribute "pull = False" to disable the pull?

Can we give a good error message if the image isn't available locally?

Reply:
I tried (1), but it proved to be harder than I thought.

Turns out my PR is misguided: the digest depends also on the registry, and docker only computes the digest when you push. Indeed, local images that you never pushed have no digest; see `docker images --digests`.

Approach (2) looks good, I implemented that.